pgmpy/models/LinearGaussianBayesianNetwork.py

```diff
@@ -407,6 +422,23 @@ def simulate(
         else:
             model = self
 
+        # Step 3: If virtual_interventions are specified, change the CPD's of intervened variables
+        if virtual_intervention != []:
+            # Step 3.1: Check if CPD's are valid
+            for cpd in virtual_intervention:
+                var = cpd.variable
+                if var not in model.nodes():
+                    raise ValueError(
+                        f"Virtual intervention provided for variable which is not in the model: {var}"
+                        f"The following nodes are present in the model: {model.nodes()}"
+                    )
+
+            for cpd in virtual_intervention:
```

Comment:
We would also need to remove all incoming edges to the variables. This currently is not a valid model as every CPD in the model needs to be `P(var | parents(var))`, however, here `var` has parents but the CPD is just `P(var)`